jwst/scripts/adjust_wcs.py

```diff
@@ -0,0 +1,251 @@
+
+    # Parse input parameters
+    parser = argparse.ArgumentParser(
+        prog="adjust_wcs",
+        description="Apply adjustments to data model's WCS"
+    )
+
+    parser.add_argument(
+        "arg0",
+        nargs='*',
+        metavar="input",
+        type=str,
+        help="Input data model file(s)."
+    )
+
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument(
+        "-u",
+        "--update",
+        action="store_true",
+        help="Update input data models."
+    )
+    group.add_argument(
+        "--suffix",
+        type=str,
+        help="Suffix for output files."
+    )
+    group.add_argument(
+        "-f",
+        "--file",
+        type=str,
+        help="Output file name."
+    )
+
+    parser.add_argument(
+        "--overwrite",
+        action="store_true",
+        help="Overwrite existing output files (ignored with -u/--update)."
+    )
+
+    parser.add_argument(
+        "-x",
+        "--expand",
+        action="store_true",
+        help="Expand wild cards in input file names (cannot be combined with "
+             "-f/--file)."
+    )
+
+    group_wcs = parser.add_argument_group(
+        title='WCS adjustment parameters'
+    )
+    group_wcs.add_argument(
+        "-r",
+        "--ra_delta",
+        type=float,
+        default=0.0,
+        help="Delta RA (longitude), degrees."
+    )
+    group_wcs.add_argument(
+        "-d",
+        "--dec_delta",
+        type=float,
+        default=0.0,
+        help="Delta DEC (latitude), degrees."
+    )
+    group_wcs.add_argument(
+        "-o",
+        "--roll_delta",
+        type=float,
+        default=0.0,
+        help="Delta roll angle, degrees."
+    )
+    group_wcs.add_argument(
+        "-s",
+        "--scale_factor",
+        type=float,
+        default=1.0,
+        help="Scale factor."
+    )
+
+    parser.add_argument(
+        "-v",
+        "--version",
+        action="version",
+        version="v{0}".format(jwst.__version__)
+    )
+
+    options = parser.parse_args()
+
+    if options.file:
+        if options.expand:
+            parser.error(
+                "argument -x/--expand: not allowed with argument -f/--file"
+            )
+        if len(options.arg0) > 1:
+            parser.error(
+                "only one input file allowed with argument -f/--file"
+            )
+
+    wcs_pars = [
+        options.ra_delta,
+        options.dec_delta,
+        options.roll_delta,
+        options.scale_factor
+    ]
+
+    if wcs_pars == [0.0, 0.0, 0.0, 1.0]:
+        logger.info(
+            "All WCS adjustment parameters ('ra_delta', 'dec_delta',"
+            "roll_delta, and scale_factor) have default values"
+            "(identical transformation)."
+        )
+        logger.info("Nothing to do. Quitting.")
+        return
+
+    if options.expand:
+        files = []
+        for f in options.arg0:
+            files.extend(glob.glob(f))
+    else:
+        files = options.arg0
```

Comment:
Is the `expand` option necessary? Isn't `glob.glob(options.arg0)` equivalent to the logic here returning a list of file names in both the case of wild cards and single file input?

Reply:
Yes, there could be multiple input files that all need to be globbed, for example:

```
adjust_wcs jw_f425_miri_*_cal.fits jw_f1080_nircam_*_tweakreg.fits
```

Yes, we could replace this with `jw_f*.fits` but allowing multiple input files enables a better/finer control control over file selection.

Reply:
In my opinion this is going too far and limiting it to one option will make the interface cleaner.